src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -5,6 +5,7 @@
 using System.Data.Common;
 using System.Diagnostics.CodeAnalysis;
 using System.Net;
+using System.Runtime.CompilerServices;
```

Comment:
Please fix PR description - remove unnecessary info from the description template, add reason why chose to do `InternalsVisibleTo`.

Reply:
fixed description.